JASP-Engine/JASP/R/correlation.R

```diff
@@ -1064,3 +979,56 @@ Correlation <- function(dataset=NULL, options, perform="run", callback=function(
 	text(0, .5, errorMessage, cex=cexText)
 
 }
+
+### Utility functions for nonparametric confidence intervals ###
+.ConcordanceFunction <- function(i, j)
+{
```

Comment:
@JohnnyDoorn this breaks for contNormal and debMiss1

Reply:
Fixed in pull request #1999